pkg/reconciler/revision/resources/deploy.go

```diff
@@ -167,7 +187,7 @@ func makeServingContainer(servingContainer corev1.Container, rev *v1.Revision) c
 func BuildPodSpec(rev *v1.Revision, containers []corev1.Container) *corev1.PodSpec {
 	pod := rev.Spec.PodSpec.DeepCopy()
 	pod.Containers = containers
-	pod.Volumes = append([]corev1.Volume{varLogVolume}, rev.Spec.Volumes...)
+	pod.Volumes = rev.Spec.Volumes
```

Comment:
I think you can just delete this line, since I think `rev.Spec.PodSpec.Volumes == rev.Spec.Volumes`, which was copied on 188.  As is, we're no longer using the copy, which seems like a potentially mild source of bugs.

Reply:
ahh good catch